app/src/main/java/org/wikipedia/page/PageProperties.kt

```diff
@@ -0,0 +1,85 @@
+package org.wikipedia.page
+
+import android.location.Location
+import android.os.Parcelable
+import kotlinx.parcelize.IgnoredOnParcel
+import kotlinx.parcelize.Parcelize
+import org.wikipedia.auth.AccountUtil.isMemberOf
```

Comment:
Please only import `import org.wikipedia.auth.AccountUtil` and use `AccountUtil.isMemberOf`